docs/Job-reference.md

```diff
@@ -4286,3 +4286,46 @@ job {
 ```
 
 (since 1.26)
+
+# Workflow Definitions
+
+## Groovy CPS DSL
+
+```
+job(type: Workflow) {
+    definition {
+        cps {
+            script(String script)
+            sandbox(boolean sandbox = true)
+        }
+    }
+}
+```
+
+Defines a Groovy CPS DSL definition.
+
+```
+def flow = '''node {
+  git url: 'https://github.com/jglick/simple-maven-project-with-tests.git'
+  def mvnHome = tool 'M3'
+  sh "${mvnHome}/bin/mvn -B verify"
+}'''
```

Comment:
The hitch, of course, is that if the flow script itself needs to use `'''` or `\` or whatever, you get into a double-escaping hell. Is it straightforward to load the flow script from a separate file? (But see my comment about `CpsScmFlowDefinition`, which is perhaps the cleaner way to do this anyway.)